paasta_tools/chronos_tools.py

```diff
@@ -956,3 +969,13 @@ def is_temporary_job(job):
     :returns: a boolean indicating if a job is a temporary job
     """
     return job['name'].startswith(TMP_JOB_IDENTIFIER)
+
+
+def chronos_services_running_here():
+    """See what chronos services are being run by a mesos-slave on this host.
+    :returns: A list of triples of (service, instance, port)"""
+
+    return mesos_services_running_here(
+        framework_filter=lambda fw: fw['name'].startswith('chronos'),
+        parse_service_instance_from_executor_id=lambda task_id: decompose_job_id(task_id.split(MESOS_TASK_SPACER)[3])
```

Comment:
true - isn't this the same for marathon too though? 

A completely different approach I could take would be to query the docker client for running containers, and then extract the service and instance name from env variables (sure would be nice to add labels). But marathon_services_running_here() was already the existing pattern so I stuck with that.